frontend/packages/integration-tests-cypress/support/selectors.ts

```diff
@@ -14,7 +14,7 @@ declare global {
 
 Cypress.Commands.add('byTestID', (selector: string) => cy.get(`[data-test="${selector}"]`));
 Cypress.Commands.add('byTestActionID', (selector: string) =>
-  cy.get(`[data-test-action="${selector}"]:not(.pf-m-disabled)`),
+  cy.get(`[data-test-action="${selector}"]:not(.pf-m-disabled)`, { timeout: 60000 }),
```

Comment:
This is still way, way too long. If you were a user, you wouldn't wait a full minute for the action to enable :) If this is happening, we want the test to fail.

Reply:
ok, timeout has been removed.